src/network/raknet/util/InetAddress.ts

```diff
@@ -0,0 +1,42 @@
+import { RemoteInfo } from "dgram";
+
+export interface InetAddressData {
+    address: string;
+    port: number;
+    family?: "IPv4" | "IPv6";
+}
+
+export default class InetAddress {
+    private readonly address: string;
+    private readonly port: number;
+    private readonly family: string;
+
+    constructor(addressInfo: InetAddressData) {
+        this.address = addressInfo.address;
+        this.port = addressInfo.port;
+        this.family = addressInfo.family ?? "IPv4";
+    }
+
+    public getAddress(): string {
+        return this.address;
+    }
+
+    public getPort(): number {
+        return this.port;
+    }
+
+    public getFamily(): string {
+        return this.family;
+    }
+
+    public toToken(): string {
```

Comment:
This should be a getter function
```ts
public get token(): string {
	return `${this.address}:${this.port}`;
}
```

Reply:
That is basically a personal preference, i don't like that much JS getters, i would instead wait for this project to grow: 
https://github.com/Exide/tombok.
But if you both want, i can switch to JS getters